### PR TITLE
feat(Switch): [switch] suppor custom open and close icon.

### PR DESCRIPTION
**Pull request by AcWrong02**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
用户可以自定义开关图标

Issue Number: #1204 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gimmyhehe — DISMISSED** [review]

**Comment by kagol** [comment]
@AcWrong02 E2E测试挂了一个，麻烦分析下哈

File: examples/sites/demos/pc/app/switch/custom-open-close-icon.spec.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { test, expect } from '@playwright/test'
+
+test('自定义开关图标', async ({ page }) => {
+  page.on('pageerror', (exception) => expect(exception).toBeNull())
+  await page.goto('switch##custom-open-close-icon')
+
+  const demo = page.locator('#custom-open-close-icon')
+  const switchBtn = demo.locator('.tiny-switch')
+
+  await expect(switchBtn.first()).toHaveAttribute()('class', 'tiny-svg')
```
Comment (gimmyhehe) [inline examples/sites/demos/pc/app/switch/custom-open-close-icon.spec.ts]: @AcWrong02 看起来是E2E用例编写有问题，第10行多出了`()`

**Comment by kagol** [comment]
@AcWrong02 E2E测试用例未通过

**Comment by AcWrong02** [comment]
> @AcWrong02 E2E测试用例未通过

应该可以了

**Review by Youyou-smiles — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Review by chenxi-20 — APPROVED** [review]

File: packages/theme/src/switch/index.less [hunk 2]
```diff
@@ -79,7 +79,10 @@
     border-color: var(--ti-switch-checked-disabled-border-color);
   }
 
-  &::after {
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (zzcr) [inline packages/theme/src/switch/index.less:85]: 这里删除&::after选择器的原因是什么呢？

Comment (AcWrong02) [reply packages/theme/src/switch/index.less:85]: 将圆圈设置成div元素了，原本实现是通过::after实现
old：
image
new：
image

**Review by zzcr — APPROVED** [review]

**Pull request merged**